src/onefetch/deps/package_parser.rs

```diff
@@ -0,0 +1,26 @@
+use crate::onefetch::error::*;
+use {regex::Regex, toml::Value};
+
+pub fn cargo(contents: &str) -> Result<usize> {
+    let parsed = contents.parse::<Value>()?;
+
+    Ok(parsed["dependencies"].as_table().unwrap().len())
+}
+
+pub fn go_modules(contents: &str) -> Result<usize> {
+    let count = Regex::new(r"v[0-9]+")?.find_iter(contents).count();
+
+    Ok(count)
+}
+
+pub fn npm(contents: &str) -> Result<usize> {
+    let parsed = json::parse(contents)?;
+
+    Ok(parsed["dependencies"].len())
```

Comment:
See comment on `fn cargo`.

Reply:
I just tried. At least for NPM, it gives a 0 by default.